core/src/main/scala/org/apache/spark/sql/row/JDBCMutableRelation.scala

```diff
@@ -441,6 +439,26 @@ case class JDBCMutableRelation(
     throw new UnsupportedOperationException()
   }
 
+  private def getDataType(column: StructField): String = {
```

Comment:
There is a method  GemFireXDBaseDialect.getJDBCType. Will that method work here?  The code looks very similar.

Reply:
Let me check. We have the dialect object there. Should work.